src/index.js

```diff
@@ -0,0 +1,71 @@
+// CHANGING CITY NAME
+let city = document.getElementById("cityName");
+let changeCity = document.getElementById("changeCity");
+const resetButton = document.getElementById("resetButton");
+
+// CHANGING THE TEMPERATURE AND LANDSCAPE
+// -- TEMP VARIABLES -- //
+const increaseTemp = document.getElementById("increaseTemp");
+const decreaseTemp = document.getElementById("decreaseTemp");
+let temp = document.getElementById("num");
+// -- LANDSCAPE VARIABLES -- //
+let sky = document.getElementById("sky");
+let garden = document.getElementById("garden");
+let selectSky = document.getElementById("selectSky");
```

Comment:
Putting the `getElementById` calls here is a bit risky. It's possible that the DOM may not have fully loaded by the time these calls are executed, which could cause the elements to sometimes not be found. This logic would be better placed `registerEventHandlers` because we know the DOM content is full ready by the time that function is called. You can still declare the variables out here if you want the global scope - just wait to initialize them.